sdk/maps/maps-timezone-rest/package.json

```diff
@@ -13,97 +13,77 @@
     "isomorphic"
   ],
   "license": "MIT",
-  "main": "dist/index.js",
-  "module": "./dist-esm/src/index.js",
-  "types": "./types/maps-timezone-rest.d.ts",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/commonjs/index.d.ts",
   "repository": "github:Azure/azure-sdk-for-js",
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "files": [
     "dist/",
-    "dist-esm/src/",
-    "types/maps-timezone-rest.d.ts",
     "README.md",
-    "LICENSE",
-    "review/*"
+    "LICENSE"
   ],
   "engines": {
     "node": ">=18.0.0"
   },
   "scripts": {
-    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
-    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build": "npm run clean && dev-tool run build-package && dev-tool run extract-api",
+    "build:browser": "dev-tool run build-package && cross-env ONLY_BROWSER=true rollup -c 2>&1",
```

Comment:
remove cross-env, maybe copy and paste from another maps package.

Reply:
grrr I could have sworn I did! Now I am just embarrassed 😄 

Thanks for catching!